tests/Backend/Mixed/SharingTest.php

```diff
@@ -1162,6 +1162,106 @@ function (array $column) {
         $this->assertCount(5, $workspaceTableData);
     }
 
+    /**
+     * @dataProvider sharingBackendData
+     * @throws ClientException
+     */
+    public function testOrganizationShareBucket($backend)
+    {
+        $this->initTestBuckets($backend);
+        $bucketId = reset($this->_bucketIds);
+
+        // first share
+        $this->_client->shareOrganizationBucket($bucketId);
+        $this->assertTrue($this->_client->isSharedBucket($bucketId));
+
+        $this->_client->unshareBucket($bucketId);
+        $this->assertFalse($this->_client->isSharedBucket($bucketId));
+
+        // sharing twice
+        $this->_client->shareOrganizationBucket($bucketId);
+
+        try {
+            $this->_client->shareOrganizationBucket($bucketId);
```

Comment:
Když to jde (a tady to podlě mě jde), tak bych používal spíš
`$this->expectedException()` a `$this->expectedExceptionMessage()`
Je to podle mě takový víc jasný, ale podle naší [Programátorské ústavy](https://keboola.atlassian.net/wiki/spaces/KI/pages/71958776/Work+Standards) § "Reviews"  je tahle připomínka zakázaná :grimacing:

Reply:
Da sa tam skontrolovat pomocou tych metod aj ten `stringCode` co to vracia ?

Reply:
Ne, protože `stringCode` je naše implementace -> https://github.com/keboola/storage-api-php-client/blob/593403cd24ed4b8fd60609c663ae57d33d55d436/src/Keboola/StorageApi/Exception.php#L11 a jsou to podle mě jenom "metadata", který slouží pro lepší strojový zpracování. Zkusím se podívat jestli by to tam nešlo doplnit

Reply:
Musela by se přetížit metoda `run` v TestCase, což by šlo udělat, ale zbytečně by to blocknulo update phpunitu :confused: